libs/core/langchain_core/output_parsers/json.py

```diff
@@ -202,19 +202,19 @@ def _diff(self, prev: Optional[Any], next: Any) -> Any:
     def parse_result(self, result: List[Generation], *, partial: bool = False) -> Any:
         text = result[0].text
         text = text.strip()
-        if partial:
```

Comment:
this is needed for streaming json, since streamed chunks of json will often be invalid. the behavior of returning None in this case is intentional